platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/LongListHeap.java

```diff
@@ -51,6 +52,10 @@
 @SuppressWarnings("unused")
 public final class LongListHeap extends AbstractLongList<AtomicLongArray> {
 
+    /** A temp buffer for reading chunk data from the file during initialization. */
+    private static final ByteBuffer TEMP_READ_BUFFER =
```

Comment:
Why static? This temp buffer is only needed, when the long list is initialized from a file. After that, it's just a waste of memory. Can the same pattern as in `LongListDisk` be used here? I mean, override `readBodyFromFileChannelOnInit()`, create a temp buffer, store it in a field, then call `super.readBodyFromFileChannelOnInit()`, then destroy the buffer and set it to null.